### ISSUE TITLE
VI-series (slotted-config) update unreliable due to MIDI stack problems

### ISSUE THREAD

**Issue by abridgewater** [abridgewater/alesisvsysex#10]
This is well covered in commit messages and code comments, but the short explanation is that we need to use `SNDRV_RAWMIDI_IOCTL_DRAIN`, which is wrapped in libalsa as `snd_rawmidi_drain()` after sending each packet to the keyboard.  This is not exposed in RtMidi, thus not exposed in python_rtmidi, thus not exposed in mido, thus not available for our use.

For the time being, we KLUDGE around the issue by sleeping for about two microseconds between each packet, which is non-optimal in several respects.

Basically, we need to find a better option for handling MIDI.
